src/components/Section.js

```diff
@@ -11,7 +11,7 @@ export default class Section {
     });
   }
 
-  addItem(element) {
-    this._container.prepend(element);
+  addCard(cardElement) {
+    this._container.appendChild(cardElement);
```

Comment:
* New cards should be added only with `prepend` to the start of the section otherwise the user doesn’t see the new card and may think that it hasn’t been created